lib/core/core.module.ts

```diff
@@ -39,7 +39,7 @@ import { ToolbarModule } from './toolbar/toolbar.module';
 import { UserInfoModule } from './userinfo/userinfo.module';
 import { ViewerModule } from './viewer/viewer.module';
 import { FormModule } from './form/form.module';
-import { SidenavLayoutModule } from './sidenav-layout/sidenav-layout.module';
+import { LayoutModule as SidenavLayoutModule } from './layout/layout.module';
```

Comment:
We should not be renaming the components internally I think.

Reply:
After discussed it on Slack with @DenisVuyka , until we export the module with the old (@deprecated) name, it is ok.